cmake/modules/AddSwift.cmake

```diff
@@ -1085,7 +1092,7 @@ function(_add_swift_library_single target name)
         # libraries are only associated with shared libraries, so add an
         # additional check for that as well.
         set(import_library ${library})
-        if(TARGET ${library})
+        if(TARGET ${library} AND NOT "${CMAKE_SYSTEM_NAME}" STREQUAL "Windows")
```

Comment:
Ah yeah I forgot to put this in the discussion point section in the PR description.
Basically, if we leave this code on Windows, I get the following error immediately when running the cmake build:

`could not find dependency 'swiftCore_IMPLIB-NOTFOUND'`
When this is commented out, everything seems to work.